amp-rnaseq_reprocessing/amp-rnaseq_reprocess-workflow/jobs/mayo/options.json

```diff
@@ -0,0 +1,13 @@
+{
+  "cluster_name": "rna-seq-reprocessing-scicomp-toil-cluster-v001",
+  "run_name": "myo",
+  "log_level": "DEBUG",
+  "retry_count": "3",
```

Comment:
Only 3 retries? Let's bump it to at least 5, maybe higher if you think appropriate